docs/General/on-the-server.rst

```diff
@@ -0,0 +1,109 @@
+..
+    SPDX-License-Identifier: GPL-3.0-or-later
+    SPDX-FileCopyrightText: 2022 louis94 <m_louis30@yahoo.com>
+
+Freeciv21 on the Server
+***********************
+
+In order to play Freeciv21 on the network, one machine needs to act as the *server*: the state of
+the game lives on the server and players can connect to play. There are many ways to operate such a
+server; this page gives an introduction to the topic.
+
+In order to run a server, you need a computer that other players can reach over the network. If
+you're planning to play at home, this can usually be any machine connected to your WiFi or local
+cabled network. Otherwise, the easiest is to rent a small server from a hosting provider. We
```

Comment:
Instead of "we", I recommend "the Longturn Community". It is technically the same thing, but gives more definition of who "we" are.

Reply:
I added a note to the first paragraph: "This page gives an introduction to the topic, guided by the experience gathered by running the [Longturn](https://longturn.net) website."

Reply:
OK